src/renderer/components/dock/create-resource.tsx

```diff
@@ -21,6 +25,39 @@ interface Props {
 @observer
 export class CreateResource extends React.Component<Props> {
   @observable error = "";
+  templates:GroupSelectOption[] = [];
+
+  componentDidMount(){
+    let rootPaths: any[]= [];
```

Comment:
Maybe `string[]`?

Reply:
No, I use {root: string, value: value} interface to handle store (built-in of user paths)  of current folder/file. 
I'll fix it.